.pylintrc

```diff
@@ -7,7 +7,7 @@ extension-pkg-whitelist=lxml
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS,.git,.venv,.idea
```

Comment:
This change is unrelated to the PR. Could you please split it into a separate commit at least?